CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ OPTION(WANT_WINMM	"Include WinMM MIDI support" OFF)
 OPTION(WANT_QT5		"Build with Qt5" OFF)
 
 
+SET(WINE_CXX_FLAGS "" CACHE STRING "Extra flags passed to wineg++")
```

Comment:
This is a configuration variable, like `WANT_VST_NOWINE`; it could be configured with `cmake-gui`. I see that conditional variables appear after configuring; perhaps this variable should be moved to `plugins/vst_base/CMakeLists.txt`.